docs/reference/configuration/uds-operator/overview.md

```diff
@@ -6,7 +6,7 @@ sidebar:
 
 ![UDS Operator Overview Flows](https://github.com/defenseunicorns/uds-core/blob/main/docs/.images/diagrams/uds-core-operator-custom-resources.svg?raw=true)
 
-The UDS Operator plays a pivotal role in managing the lifecycle of UDS Package Custom Resources (CRs) along with their associated resources like NetworkPolicies and Istio VirtualServices. Leveraging [Pepr](https://github.com/defenseunicorns/pepr), the operator binds watch operations to the [enqueue and reconciler](https://docs.pepr.dev/v0.42.0/user-guide/actions/reconcile/), taking on several key responsibilities for UDS Packages and exemptions:
+The UDS Operator plays a pivotal role in managing the lifecycle of UDS Package Custom Resources (CRs) along with their associated resources like NetworkPolicies and Istio VirtualServices. Leveraging [Pepr](https://github.com/defenseunicorns/pepr), the operator binds watch operations to the [enqueue and reconciler](https://docs.pepr.dev/v0.52.2/user-guide/actions/reconcile/), taking on several key responsibilities for UDS Packages and exemptions:
```

Comment:
@cmwylie19 is there a way for us to link a more persistent current doc here? It seems like the `current` path works for the docs homepage but not for sub-pages.

Otherwise we could have renovate track this I guess.